corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -916,15 +916,18 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
     FileEntry.prototype.constructor = EntrySingleAnswer;
     FileEntry.prototype.onAnswerChange = function (newValue) {
         var self = this;
-        if (newValue !== constants.NO_ANSWER) {
+        if (self.answer() && self.answer().match(/([a-z0-9]{8}-)([a-z0-9]{4}-){3}([a-z0-9]{12}).[a-z]{3}/)) {
+            return
+        }
+        if (newValue !== constants.NO_ANSWER && newValue !== "") {
             var $input = $('#' + self.entryId);
             self.answer(newValue.replace(constants.FILE_PREFIX, ""));
             self.file($input[0].files[0]);
+            this.question.onchange();
         } else {
-            self.answer(newValue);
             self.file(null);
+            self.answer(constants.NO_ANSWER);
         }
```

Comment:
does setting to null/NO_ANSWER need to call `this.question.onchange();` ?

Reply:
I'm avoiding that here because `onchange` ends up calling `answerQuestion` which sends a request to formplayer that will fail since there is no file being sent.